src/main/scala/mesosphere/marathon/ZookeeperConf.scala

```diff
@@ -17,11 +17,11 @@ trait ZookeeperConf extends ScallopConf {
 
   lazy val zooKeeperTimeout = opt[Long]("zk_timeout",
```

Comment:
What's the difference between these two timeouts?

We should add more info to the descriptions.